### ISSUE TITLE
bug: app shows blank screen on bootstrap if network connection is corrupted

### ISSUE THREAD

**Issue by 0x-r4bbit** [machinelabs/machinelabs#569]
```
vendor.c4029c5f8a6c86c40972.bundle.js:1 ERROR Error: Uncaught (in promise): Error: A network error (such as timeout, interrupted connection or unreachable host) has occurred.
    at c (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at c (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at vendor.c4029c5f8a6c86c40972.bundle.js:1
    at t.invokeTask (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at Object.onInvokeTask (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at t.invokeTask (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at r.runTask (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at i (vendor.c4029c5f8a6c86c40972.bundle.js:1)
    at <anonymous>
```

We should handle such an error and at least show sth. like a "something went wrong" message

**Comment by Jefiozie** [comment]
@PascalPrecht  just to verify the correct scenario this is when the application is loaded right?

**Comment by PascalPrecht** [comment]
@Jefiozie Yes, it's a little bit of a tricky scenario. Basically what happened to me was that I opened machinelabs, then the network connection was interupted, which caused `bootstrap()` to fail. and I got a blank screen. So yes, one can say that the application is loaded, but the execution failed on bootstrap.

Does that make sense?

**Comment by Jefiozie** [comment]
I see, what we could do is : ```<ml-app> loading your page if it takes to long maybe something has gone wrong```. The textual part will go away when bootstrapping is finished.
Second we can do maybe something in the error handling of the bootstrap. We could add a message dynamically but if the loading of the files is not far enough we still won't see anything there.

**Comment by PascalPrecht** [comment]
Right, so everything that's put between the root component's tags `<ml-app>` is considered "light DOM" and will be replace with the component's view ("shadow dom"). 

But that's gonna show up every time you bootstrap the app until Angular is done bootstrapping. So this is really rather useful to show like a big loading splash screen or something.

For the case that bootstrap() fails (for whatever reason) we should really rather show some screen that says "Something went wrong" and then a button to reload the app. Maybe even a link that points people to open an issue.
